apps/src/aichat/views/ChatMessage.tsx

```diff
@@ -119,24 +141,30 @@ const displayModelUpdateMessage = (
 };
 
 const displayErrorMessage = (
```

Comment:
Rename this since it's being used to display non-error notifications?

Reply:
Done!